raiden/accounts.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser('~')
+    if home == '~':  # Could not expand user path
+        return None
+    datadir = None
```

Comment:
`datadir` is being attributed below or an exception is raised. No need for this line?